htsget.md

```diff
@@ -165,7 +165,9 @@ The server SHOULD reply with an `UnsupportedFormat` error if the requested forma
 `referenceName` 
 _optional_
 </td><td>
-The reference sequence name, for example "chr1", "1", or "chrX". If unspecified, all reads (mapped and unmapped) are returned. [^b]
+The reference sequence name, for example "chr1", "1", or "chrX". If unspecified, all records are returned regardless of position.
+  
+For the reads endpoint: if "\*", unplaced unmapped reads are returned. If unspecified, all reads (mapped and unmapped) are returned. (*Unplaced unmapped* reads are the subset of unmapped reads completely without location information, i.e., with RNAME and POS field values of "*" and 0 respectively. See the [SAM specification](http://samtools.github.io/hts-specs/SAMv1.pdf) for details of placed and unplaced unmapped reads.)
```

Comment:
Sorry if I missed this in the discussion, but why the backslash in "\*"?

Reply:
Aha. Markdown, that's why. My bad; I should be some more coffee I think...

Reply:
Back in the first round of review, @magicDGS figured * needed escaping to avoid it being interpreted as markdown emphasis/bold/whatever. I've just tested this in Jekyll's kramdown and GFM and both appear happy either way. ¯\\\_(ツ)\_/¯

Reply:
Interesting @jmarshall; I just realized in my markdown editor. By the way, there is still an unescaped `*`, which also renders in pure markdown as emphasis...

Reply:
There is indeed, later in that line. I've now tried commenting out `let html_no_rendering=1` in my _.vimrc_, and see that vim also highlights that wrongly without the backslash. So I'm now in @magicDGS's camp and agree that the lowest common denominator is that we should escape all of these literal `"\*"` characters :smile: